README.md

```diff
@@ -249,20 +233,3 @@ A comparison of different data formats in each stage of ML development cycle.
 | Exploration         | Fast  | Slow          | Fast       | Slow     | Fast     | Decent    |
 | Infra Support       | Rich  | Rich          | Decent     | Limited  | Rich     | Rich      |
 
-## Community Highlights
```

Comment:
this and next section we should probably make them pages in the doc site, so remove for now.